docs/src/quickstart.md

```diff
@@ -1,10 +1,9 @@
 # Quickstart
 
-This section describes how to test a PostgreSQL cluster on your laptop/computer
-using CloudNativePG on a local Kubernetes cluster in [Kind](https://kind.sigs.k8s.io/) or
+This section guides you through testing a PostgreSQL cluster on your local machine by deploying CloudNativePG on a local Kubernetes cluster using either [Kind](https://kind.sigs.k8s.io/) or
 [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/).
 
-!!! Warning
+> [!WARNING]
```

Comment:
These changes are not compatible with the MkDocs format for admonitions.
See https://squidfunk.github.io/mkdocs-material/reference/admonitions/

Reply:
Thanks for your feedback. I appreciate it.🙏